packages/components/tabs/src/tab-list.tsx

```diff
@@ -9,7 +9,12 @@ export interface TabListProps extends HTMLUIProps {}
 
 export const TabList = forwardRef<TabListProps, "div">(
   ({ className, ...rest }, ref) => {
-    const { focusedIndex, orientation, tabListProps, styles } = useTabsContext()
+    const {
+      focusedIndex,
+      orientation = "horizontal",
```

Comment:
Please remove default value.

Reply:
Thanks for pointing that out!
I've removed the default value of `orientation`.